src/doc_builder/commands/build.py

```diff
@@ -51,6 +107,46 @@ def build_command(args):
     if "pr_" not in version and os.path.isfile(os.path.join(package_doc_path, "_versions.yml")):
         update_versions_file(os.path.join(args.build_dir, args.library_name), version)
 
+    # If asked, convert the MDX files into HTML files.
+    if args.html:
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            tmp_dir = Path(tmp_dir)
+            # Copy everything in a tmp dir
+            shutil.copytree(kit_folder, tmp_dir / "kit")
+            if (tmp_dir / "kit" / "src" / "routes" / args.language).is_dir():
+                # Remove the current MDX dir if it exists.
+                shutil.rmtree(tmp_dir / "kit" / "src" / "routes" / args.language)
+            shutil.copytree(output_path, tmp_dir / "kit" / "src" / "routes" / args.language)
+
+            # Build doc with node
+            working_dir = str(tmp_dir / "kit")
+            print("Installing node dependencies")
```

Comment:
Feel free to change this statement to anything else if `npm ci` does not do that :-)